dsst_etl/db.py

```diff
@@ -1,21 +1,30 @@
+from sqlalchemy import inspect
 from sqlalchemy.orm import sessionmaker
 from sqlalchemy_utils import create_database, database_exists
 
-from dsst_etl import get_db_engine, logger
+from dsst_etl import logger
 
 from .models import Base
 
 
-def get_db_session(is_test=False):
-    engine = get_db_engine(is_test)
+def get_db_session(engine):
     Session = sessionmaker(bind=engine)
     return Session()
 
 
-def init_db(is_test=False):
-    engine = get_db_engine(is_test)
+def init_db(engine):
+    logger.info("Initializing database")
 
     if not database_exists(engine.url):
-        logger.info("Creating database.....")
+        logger.info("Creating database...")
         create_database(engine.url)
+        logger.info("Database created")
+
+    logger.info("Creating tables...")
```

Comment:
Now that we have alembic working should we remove this? If the database schema is not initialised with `alembic upgrade head` alembic migrations won't work. I think modifying the developer readme to suggest doing that and defensively raising an error here would be best but I am open to suggestions.

Reply:
yes, it is just for run in unit-test for create database in case no database created, but let's me modifying in README and move those into unit-test folder

Reply:
yes, just update the CI for this initial database.